terraform/cloudwatch.tf

```diff
@@ -59,20 +59,14 @@ resource "aws_cloudwatch_event_target" "root_server_import" {
   role_arn  = aws_iam_role.ecs_events.arn
 
   ecs_target {
+    launch_type         = "FARGATE"
     task_count          = 1
     task_definition_arn = aws_ecs_task_definition.tomato_root_server_import.arn
+
+    network_configuration {
+      security_groups  = [aws_security_group.cluster.id]
+      subnets          = [aws_subnet.public_a.id, aws_subnet.public_b.id]
+      assign_public_ip = true
```

Comment:
ecs targets shoudn't have public ips

Reply:
they're behind a load balancer, you don't hit them directly

Reply:
what is associating the tasks with the load balancer? are you doing that manually?

Reply:
It gets attached here https://github.com/bmlt-enabled/tomato/blob/183276f4145f5d7214597153ce8e60aca9097ef8/terraform/ecs-service.tf#L222